app/Http/ViewHelpers/Employee/EmployeeHierarchyViewHelper.php

```diff
@@ -27,26 +35,15 @@ public static function search(Company $company, Employee $employee, string $crit
                     ->orWhere('email', 'LIKE', '%'.$criteria.'%');
             })
             ->where('id', '!=', $employee->id)
+            ->whereNotIn('id', $existingManagersForTheEmployee->merge($existingDirectReportsForTheEmployee))
```

Comment:
Even after reading the doc, I don't understand what `merge` does here.

Reply:
both collection contains a list of ids.
`merge` will merge the 2 collections.